aip-console-tools-cli/src/test/java/com/castsoftware/aip/console/tools/OnboardApplicationFastScanCommandIntegrationTest.java

```diff
@@ -57,7 +57,7 @@
 @ActiveProfiles(TestConstants.PROFILE_INTEGRATION_TEST)
 public class OnboardApplicationFastScanCommandIntegrationTest extends AipConsoleToolsCliBaseTest {
```

Comment:
Also rename https://cast-products.atlassian.net/browse/WEBI-16170 to FastScanIntegrationTes
See OnbordApplication if such name already exist

Reply:
Changes done